packages/grid/x-data-grid/src/hooks/features/clipboard/useGridClipboard.ts

```diff
@@ -26,15 +26,15 @@ function writeToClipboardPolyfill(data: string) {
 }
 
 function hasNativeSelection(element: HTMLInputElement) {
-  if (window.getSelection()?.toString() !== '') {
```

Comment:
@cherniavskii I think that this wasn't exactly correct. Per https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

>  If the object accessed or function called is undefined or null, it returns undefined instead of throwing an error.

So when the selection returns null, the chaining returns `undefined` which we then compare to `''`. it's different, so say there is a selection. But there is no selection in this case, in theory, per https://developer.mozilla.org/en-US/docs/Web/API/Window/getSelection.

> When called on an [<iframe>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe) that is not displayed (e.g., where display: none is set) Firefox will return null